compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -4134,9 +4143,9 @@ private void startContextIfRequired(ParserRuleContext currentCtx) {
             case TYPE_DESC_IN_EXPRESSION:
             case TYPE_DESC_IN_STREAM_TYPE_DESC:
             case TYPE_DESC_IN_PARENTHESIS:
-            case TYPE_DESC_IN_TUPLE:
             case TYPE_DESC_IN_SERVICE:
             case TYPE_DESC_IN_PATH_PARAM:
+            case TUPLE_MEMBERS:
```

Comment:
Shall we move this out of the `// Contexts that expect a type` section as this is not relevant? 
you can add this after the `case BRACED_EXPRESSION` above.